.github/ISSUE_TEMPLATE/bug_report.yaml

```
@@ -93,6 +93,7 @@ body:
       - extension/encoding/zipkinencoding
       - extension/headerssetter
       - extension/healthcheck
+      - extension/healthcheckextensionv2
```

Sounds like it should be:
```suggestion
      - extension/healthcheckv2
```